components/Ink/src/private/MDCInkLayer.m

```diff
@@ -379,16 +432,40 @@ - (void)exit:(BOOL)animated {
   [self addAnimation:_backgroundOpacityAnim forKey:kInkLayerBackgroundOpacityAnim];
 }
 
+- (void)clearAnimations {
```

Comment:
Yeah removeAllAnimations is a method on CALayer already so it would be a conflict. [CALayer - removeAllAnimations](https://developer.apple.com/reference/quartzcore/calayer/1410810-removeallanimations?language=objc)